Configurations/unix-Makefile.tmpl

```diff
@@ -1184,12 +1184,10 @@ reconfigure reconf:
 
       if ($args{src} =~ /\.html$/) {
 	  my $title = basename($args{src}, ".html");
+          my $pod = $args{generator}->[0];
 	  return <<"EOF";
-$args{src}: $args{generator}->[0]
-	pod2html "--podroot=\$(SRCDIR)/doc" --htmldir=.. \\
-		 --podpath=man1:man3:man5:man7 "--infile=\$<" "--title=$title" \\
-	| \$(PERL) -pe 's|href="http://man\\.he\\.net/(man\\d/[^"]+)(?:\\.html)?"|href="../\$1.html|g;' \\
-	> \$\@
+$args{src}: "$pod"
+	\$(PERL) \$(SRCDIR)/util/mkpod2html.pl -i "$pod" -o \$\@ -t "$title" -r "\$(SRCDIR)/doc"
```

Comment:
So, er, better not to have quotes around `$pod` here in this .tmpl, as `make` uses them literally, i.e. the Travis error I showed indicates that it's looking for `"../_srcdist/doc/man1/CA.pl.pod"` literally, quotes included as part of the file name.

Reply:
Okay, fixup commit removed them.  I think it's a likely to be a problem for us when someone uses a path with spaces in it.  I would rather we just say that's not supported on Unix but oh well.